estate/models/model_real_estate_property_offer.xml

```diff
@@ -53,4 +53,17 @@
         <field name="required">True</field>
         <field name="on_delete">restrict</field>
     </record>
+
+    <record id="action_real_estate_property_offer_accept_offer" model="ir.actions.server">
```

Comment:
Instead/on addition to a button, it could make sense to *bind* this as well, so that it is available from the gear